autoscheduler/frontend/urls.py

```diff
@@ -4,4 +4,4 @@
 
```

Comment:
Note to self: This file's gonna change when we rebase onto `frontend/master` again.